src/deploy-cromwell-on-azure/Configuration.cs

```diff
@@ -72,6 +83,13 @@ public abstract class UserAccessibleConfiguration
         public string BlobxferImageName { get; set; } = null;
         public bool? DisableBatchNodesPublicIpAddress { get; set; } = null;
         public bool? KeepSshPortOpen { get; set; } = null;
+        public bool DebugLogging { get; set; } = false;
+        public bool? ProvisionPostgreSqlOnAzure { get; set; } = null;
+        public string PostgreSqlServerName { get; set; }
+        public bool UsePostgreSqlSingleServer { get; set; } = false;
+        public string KeyVaultName { get; set; }
+        // Temporary workaround until I can get Azure Graph RBAC client working. 
```

Comment:
Are we going to release with this workaround? If yes, UserObjectId parameter should be documented.

Reply:
Most likely, documented for now.